### ISSUE TITLE
New houses alarm bug

### ISSUE THREAD

**Issue by Chevu337** [echoroleplay/feedback#191]
**Description:**  **So in short, the bug occurs in all interiors except for south-side interiors, after you complete the hack successfully when you cross a certain imaginary line in the house alarm still goes off and every time your cross that place (depending on the interior) alarm goes off, It feels like doing the hack successfully doesn't actually disable that imaginary line from triggering the alarm.
I also tried crouching, walking in different walk styles, and also running for some reason and I still get the same result.**

**Reproduction Steps:**
1. Complete the hack successfully.
2. Walking deeper into the house alarm triggers no matter what.

**Expected Behavior:** Alarm not going off and no one gets notified.

**Actual Behavior:** I explained mostly in the description but the alarm goes off after doing the hack successfully.

**Screenshot(s)/Video(s):** If it's not obvious I can make a video.

**Comment by SurgeGit** [comment]
Yeah I should have mentioned this in my message actually. It also goes off in the exact same spot which made me think it is unintentional

**Comment by FlawwsX** [comment]
I am not able to reproduce this, can you provide a video, @Chevu337?
